travel-agency-service/src/main/java/cz/muni/fi/pa165/bluebat/service/TripServiceImpl.java

```diff
@@ -16,13 +17,14 @@
  * @author Ondrej Vaca
  */
 @Service
-@Transactional
 public class TripServiceImpl implements TripService{
 
     private final TripDao tripDao;
 
     private PriceService priceService;
 
+    private ExcursionService excursionService;
+
```

Comment:
Proč jsi to přidal, když to nikde nevyužíváš ani to nenastavuješ v konstruktoru?